lib/ansible/parsing/vault/ciphers.py

```diff
@@ -0,0 +1,238 @@
+# (c) 2014, James Tanner <tanner.jc@gmail.com>
+# (c) 2016, Adrian Likins <alikins@redhat.com>
+# (c) 2016 Toshio Kuratomi <tkuratomi@ansible.com>
+# (c) 2023, Thomas Ziegler <thomas.ziegler.pa@gmail.com>
```

Comment:
I don't think rewrites/refactoring usually warrant a copyright addition (same for the other files). These would be more applicable on a PR containing feature work.